src/kibana/plugins/visualize/editor/sidebar.html

```diff
@@ -1,7 +1,6 @@
 <div class="sidebar-container">
-  <form class="sidebar-list"
-        ng-class="{'hide-errors': hideErrors}"
-        ng-submit="visualizeEditor.$invalid ? dontApply() : stageEditableVis()"
+  <form class="sidebar-list in-time-errors"
+        ng-submit="stageEditableVis()"
```

Comment:
Is it safe to call `stageEditableVis()` when the vis is invalid? I don't think it is.